Full update to FAQ (#449)
status-im/status.im: source/technical/FAQs.md

 
### What happens if a government tries to block or censor Status?
Status messenger does not use one central server to and from which communication can be blocked. Instead Status uses a distributed network of mail servers. This ensures resilience of the network. If a mail server is shut down you can connect to another mail server or start [running your own](https://discuss.status.im/t/get-your-node-running/1488/4?u=hester). However, Status and mail servers in the network still rely on having an Internet connection. If the Internet is shut down through telecom providers you will not be able to use Status.

### How does Status connect to the blockchain? ?

> Status is packaged with go-ethereum, which connects to the Ethereum network. All you need to do to connect to Ethereum is run the Status app! Note that only the wallet and browser interact with the Ethereum blockchain. Messages are not stored on the blockchain.

Reviewer: ```suggestion
Status is implemented on top of [`go-ethereum`](https://github.com/ethereum/go-ethereum), which connects to the Ethereum network via [`devp2p`](https://github.com/ethereum/devp2p). All you need to do to connect to Ethereum is run the Status app! Note that only the wallet and browser interact with the Ethereum blockchain. Messages are not stored on the blockchain.
```

### Can I run go-ethereum myself or on a server?
We are committed to getting light nodes working on resource restricted devices. This would allow you to connect to Ethereum from your mobile phone or IoT device directly, no server required. More importantly, it would increase accessibility and resilience of the network overall. For more information, please check out https://nimbus.team/.

> If you want to run go-ethereum now, you can set up a node on a Linux device and configure the Status app to connect to this node. You can find more information [here](https://discuss.status.im/t/get-your-node-running/).

Reviewer: ```suggestion
If you want to run [`status-go`](https://github.com/status-im/status-go) now, you can set up a node on a Linux device and configure the Status app to connect to this node. You can find more information [here](https://discuss.status.im/t/get-your-node-running/).
```

Reviewer: Also, I'm not sure linking to Discourse post makes sense. That's a wall of text most people won't read that and also it's mostly relevant to CCs.

The best place to start is: https://github.com/status-im/status-go/blob/develop/MAILSERVER.md

Author: Updated the link. Would want to update again when we have a final version of the manual.

Reviewer: Definitely.

### On what devices can I use Status?
Status is tested to run on Android 7+, and iOS 13+.

### Does Status also run on desktop?
Not at the moment. Our code base is designed so that Status can run on either mobile or desktop devices. Development of Status desktop is currently paused while the team focuses on the mobile app and other infrastructure projects.

### What languages do you support?

> Status is currently available in 7 languages — English, français, italiano,中文, 한국어, 日本語, اُردُو & русский! 

Reviewer: ```suggestion
Status is currently available in 7 languages: English, français, italiano,中文, 한국어, 日本語, اُردُو & русский! 
```

### Why do I have to pay a network fee for transactions?
The network fee, also known as “gas,” covers the computational cost of executing an action on the Ethereum blockchain. It’s standard to the network. No changes can be made to the Ethereum blockchain, including transactions between users, without the expenditure of gas. The Status wallet allows you to adjust the network fee with each transaction. A higher fee will improve the processing speed for your transaction.

### How and to what addresses can I send assets?
Status wallet allows you to send ETH and ERC20 (e.g. OMG, BNB, BAT or DAI) to wallets that accept these currencies. To send assets go to `Wallet` > `Send icon [↗]`. Alternatively go to `Wallet` > `Account card` > `Send`

Then `Enter Amount` > `Select token` > `Select from account` > `Choose recipient`

You can recognize wallets that accept ETH and Ethereum based tokens by the address format. The address will start with 0x… Status wallet checks for invalid addresses. Best practice is to always triple check the address manually before signing a transaction. 

> 

Reviewer 2: I'd like to add this question:

### I recovered my account and now I only see one wallet, though I created more. Did I lose my funds?
No, they’re not lost. If you generated additional wallets from within Status, and you later recover your account, you will need to manually recover the additional wallets by following the same steps you took to create them. From the wallet, you will Add account > Add an account. Rather than generating a brand new wallet, Status will recuperate each additional wallet you created in order, and your assets will be recovered with them.

In the future, we’ll ensure all of your Status wallets are recovered automatically.

Author: Added with an update to refer to `accounts` instead of `wallets` for consistency.
